src/plugins/hetero/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -32,9 +33,11 @@ INSTANTIATE_TEST_SUITE_P(smoke_Hetero_BehaviorTests,
                                             ::testing::ValuesIn(hetero_properties)),
                          OVPropertiesTests::getTestCaseName);
 
+static std::string (*NULL_getTestCaseName)(const testing::TestParamInfo<std::string>& info) = NULL;
 INSTANTIATE_TEST_SUITE_P(smoke_HeteroOVGetMetricPropsTest,
                          OVGetMetricPropsTest,
-                         ::testing::Values(ov::test::utils::DEVICE_HETERO));
+                         ::testing::Values(ov::test::utils::DEVICE_HETERO),
+                         MARK_MANDATORY_API_FOR_HW_DEVICE(NULL_getTestCaseName));
```

Comment:
Roll back the changes in this file and skip following test case in https://github.com/openvinotoolkit/openvino/blob/4b6d2c7dac1b1594cfa5ad4dcd602dedf34f7377/src/plugins/hetero/tests/functional/skip_tests_config.cpp#L16
smoke_HeteroOVGetMetricPropsTest/OVGetMetricPropsTest.GetMetricAndPrintNoThrow_AVAILABLE_DEVICES